fast64_internal/oot/cutscene/importer/classes.py

```diff
@@ -200,6 +206,9 @@ def getCutsceneList(self):
             # if it's none then there's no cutscene in the file
             return None
 
+        if len(parsedCutscenes) == 0:
+            raise PluginError("ERROR: No cutscene was found.")
```

Comment:
Is this intended for scene imports as well? Deku tree inside won´t import with Cutscene importing on now.

Reply:
I tested and it works properly, though it won't work if you already imported that cutscene before, which might be a concern so I changed it to make it print a warning instead of skipping the import entirely